src/clib-install.c

```diff
@@ -343,8 +343,10 @@ static int install_package(const char *slug) {
 static int install_packages(int n, char *pkgs[]) {
   for (int i = 0; i < n; i++) {
     debug(&debugger, "install %s (%d)", pkgs[i], i);
-    if (-1 == install_package(pkgs[i]))
+    if (-1 == install_package(pkgs[i])) {
+      logger_error("error", "Unable to install package");
```

Comment:
Should we tell the user which package could not be installed?

```suggestion
      logger_error("error", "Unable to install package %s", pkgs[i]);
```

Reply:
Oh, I forgot that, thanks!